src/common/util.c

```diff
@@ -338,3 +339,13 @@ char *oscap_path_join(const char *path1, const char *path2)
 	joined_path[joined_path_len] = '\0';
 	return joined_path;
 }
+
+#ifdef _WIN32
+char *oscap_wstr_to_str(wchar_t *wstr)
```

Comment:
wstr and wchar have a ton of definitions, I suggest to prefix this with win to make it clear this is windows only. wchar can have different widths depending on the OS and compiler.

Reply:
+1